lib/cc/engine/flog.rb

```diff
@@ -90,10 +89,11 @@ def run
         Dir.chdir dir do
           flog.flog(*files)
 
-          flog.each_by_score flog.threshold do |name, score, call_list|
+          flog.each_by_score do |name, score, call_list|
             location = flog.method_locations[name]
 
             next unless location # XXX#main is location-less, skip for now
+            next unless score > config["max_score"]
```

Comment:
So I'm the kid in the 3rd grade who when taught "clockwise" and "counter-clockwise" asked "if I'm looking at the clock or if I _am_ the clock"... So, calibrate accordingly.

I'm having a hard time with calling this `max_score` when it is clearly used as a floor function.